viewer/viewer.js

```diff
@@ -1968,10 +1968,10 @@ app.get(
 );
 
 // cyberchef apis -------------------------------------------------------------
-app.get('/cyberchef.html', express.static( // cyberchef client file endpoint
+app.get('/cyberchef.html', [ArkimeUtil.missingResource, cyberchefCspHeader], express.static( // cyberchef client file endpoint
```

Comment:
Thanks! This doesn't seem right either since missingResource returns an error.
Maybe ```app.get('/cyberchef.html', [cyberchefCspHeader], express.static(``` and the missingResource still stays on the end?